CMakeLists.txt

```diff
@@ -21,7 +21,7 @@ if(BENCHMARK_FORCE_WERROR)
   set(BENCHMARK_ENABLE_WERROR ON)
 endif(BENCHMARK_FORCE_WERROR)
 
-if(NOT MSVC)
+if(NOT (MSVC OR CMAKE_CXX_SIMULATE_ID STREQUAL "MSVC"))
```

Comment:
I'm not sure about this one, is this unsupported generally on windows (exluding mingw), or is it a specific problem with MSVC (and clang-cl)?

Reply:
i don't know, but i'm quite comfortable with windows-like being only 64-bit at this point.